src/util/linkify.js

```diff
@@ -12,8 +12,9 @@ const createLink = options => href => {
 	const target = options.target || '';
 	const targetAttr = `target="${target}"`;
 	const relAttr = target === '_blank' ? 'rel="noopener noreferrer"' : '';
+	const link = (href.search('www.') === 0) ? `http://${href}` : href;
```

Comment:
You are right but it also matches [`https`, `ws` `ftp` and `//`](https://github.com/kevva/url-regex/blob/master/test.js#L49-L61) at the beginning of the string. I'm worried that we might miss a case if we test for protocol. it might be better to go back to our own regex expression.